### DISCUSSION TITLE
[Application Development] <Devolpment on OP Sepolia Testnet>

### DISCUSSION THREAD

**Discussion by bjmvercelli** [ethereum-optimism/developers#403] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hello, how you guys doing?

I'm a web2 web developer trying to get into web3 and i'm willing to start on OP Sepolia Testnet.

In this scenario, i'd like to practice by creating my own code to interact with contracts, just like metamask does...

May i have some resourcers recommendations? Such as SDK's, communities, tutorials, etc.

Thanks

**Comment by imtipi** [comment]
hi,welcome to the web3
for the frontend library:web3.js ether.js https://github.com/ethereum-optimism/scaffold-op (start more easily)
docs about optimism:https://docs.optimism.io/
for web3 tutorial i recommend this:https://university.alchemy.com/overview/ethereum
also looking for other's reply,for now i only know these
